File: app/src/main/java/com/coderstory/FTool/plugins/start.java
package com.coderstory.FTool.plugins;


import com.coderstory.FTool.module.Hooks;
import com.coderstory.FTool.module.isEnable;

import de.robv.android.xposed.IXposedHookInitPackageResources;
import de.robv.android.xposed.IXposedHookLoadPackage;
import de.robv.android.xposed.IXposedHookZygoteInit;
import de.robv.android.xposed.callbacks.XC_InitPackageResources;
import de.robv.android.xposed.callbacks.XC_LoadPackage;


public class start implements IXposedHookZygoteInit, IXposedHookLoadPackage, IXposedHookInitPackageResources {
    @Override
    public void handleInitPackageResources(XC_InitPackageResources.InitPackageResourcesParam resparam) throws Throwable {


    }
    @Override
    public void handleLoadPackage(XC_LoadPackage.LoadPackageParam lpparam) throws Throwable {


            new Hooks().handleLoadPackage(lpparam);
            new isEnable().handleLoadPackage(lpparam);

    }
    @Override
    public void initZygote(StartupParam startupParam) throws Throwable {

    }
}

File: app/src/main/java/com/coderstory/flyme10/xposed/Start.kt
package com.coderstory.flyme10.xposed

import com.coderstory.flyme10.patchModule.FuckAd
import com.coderstory.flyme10.patchModule.HideApp
import com.coderstory.flyme10.patchModule.IsEnable
import com.coderstory.flyme10.patchModule.Others
import com.coderstory.flyme10.patchModule.SystemUi
import com.coderstory.flyme10.patchModule.ThemePatcher
import com.coderstory.flyme10.tools.XposedHelper
import de.robv.android.xposed.IXposedHookInitPackageResources
import de.robv.android.xposed.IXposedHookLoadPackage
import de.robv.android.xposed.IXposedHookZygoteInit
import de.robv.android.xposed.IXposedHookZygoteInit.StartupParam
import de.robv.android.xposed.XposedBridge
import de.robv.android.xposed.callbacks.XC_InitPackageResources.InitPackageResourcesParam
import de.robv.android.xposed.callbacks.XC_LoadPackage.LoadPackageParam

class Start : XposedHelper(), IXposedHookZygoteInit, IXposedHookLoadPackage,
    IXposedHookInitPackageResources {
    override fun handleInitPackageResources(resparam: InitPackageResourcesParam) {
        Others().handleInitPackageResources(resparam)
        SystemUi().handleInitPackageResources(resparam)
        ThemePatcher().handleInitPackageResources(resparam)
    }

    override fun handleLoadPackage(lpparam: LoadPackageParam) {
        if (json.isEmpty()) {
            initJson(lpparam)
        }
        IsEnable().handleLoadPackage(lpparam)
        HideApp().handleLoadPackage(lpparam)
        Others().handleLoadPackage(lpparam)
        ThemePatcher().handleLoadPackage(lpparam)
        FuckAd().handleLoadPackage(lpparam)
        SystemUi().handleLoadPackage(lpparam)
    }

    override fun initZygote(startupParam: StartupParam) {
        XposedBridge.log("Flyme助手已加载")
        SystemUi().initZygote(startupParam)
    }
}